### ISSUE TITLE
Migrate to Electron 5.0

### ISSUE THREAD

**Issue by contractormario** [electron-react-boilerplate/electron-react-boilerplate#2165]
Electron 5.0 should be out any day now. Chrome 73 would be nice to have.

**Comment by supercoptech** [comment]
Nobody cares?

**Comment by praisethemoon** [comment]
In my own application, the only I had to dealwith is a runtime error in `app.html` stating that `process` is not defined. Turns out Electron has disabled a parameter that was previously on by default which is `nodeIntegration`. Just setting it to on solved the issue for me.

I would love if this repo officially migrates to v5.

References:
- https://github.com/electron/electron/issues/18139
- https://stackoverflow.com/questions/55093700/electron-5-0-0-uncaught-referenceerror-require-is-not-defined

Cheers everyone.

**Comment by videahealth-jeannei** [comment]
@praisethemoon isn't that solution because of the security concerns around it?

* https://electronjs.org/docs/tutorial/security#2-do-not-enable-nodejs-integration-for-remote-content

**Comment by citrusjunoss** [comment]
> @praisethemoon并不是最好的解决方案，因为它周围的安全问题？
> 
> * https://electronjs.org/docs/tutorial/security#2-do-not-enable-nodejs-integration-for-remote-content



> @praisethemoon isn't that solution not the best because of the security concerns around it?
> 
> * https://electronjs.org/docs/tutorial/security#2-do-not-enable-nodejs-integration-for-remote-content

Indeed, the solution has security issues，
Is there a better solution?

**Comment by citrusjunoss** [comment]
> @praisethemoon isn't that solution not the best because of the security concerns around it?
> 
> * https://electronjs.org/docs/tutorial/security#2-do-not-enable-nodejs-integration-for-remote-content

Indeed, the solution has security issues，
Is there a better solution?

**Comment by austinnichols101** [comment]
Version 3.0 is officially "end-of-life".  Version 7.0 is currently at Beta 4.
```
Currently supported versions:
6.x.y
5.x.y
4.x.y
```

Ref: https://electronjs.org/docs/tutorial/support#supported-versions

**Comment by austinnichols101** [comment]
Version 7 released.
```
Currently supported versions:
7.x.y
6.x.y
5.x.y
```

**Comment by nguyen-tam** [comment]
any solution without nodeIntegration = true?

**Comment by amilajack** [comment]
Fixed in #2325
